Jenkinsfile

```
@@ -0,0 +1,71 @@
+pipeline {
+  agent { label 'linux' }
+
+  options {
+    disableConcurrentBuilds()
+    /* checkout logos repo to use site builder */
+    checkoutToSubdirectory('src/')
+    /* manage how many builds we keep */
+    buildDiscarder(logRotator(
+      numToKeepStr: '20',
+      daysToKeepStr: '30',
+    ))
+  }
+
+  environment {
+    /* checkout the source code, use local folder */
```

No, that's not a good comment. A good comment explains WHY you are doing something, not WHAT you are doing.
Look at this:
```js
  /* Make request to API */
  api.request()
```
How is this comment useful in any way? It is not. It provides no additional information.

The commend we need is more like:
```suggestion
    /* Mode of logos-site-builder for copying site source from already checked out repo.
     * TODO: Avoid copying anything at all, make checkout site of into `docs` folder work. */
```